pkg/whitesource/configHelper.go

```diff
@@ -171,8 +171,15 @@ func (c *ConfigOptions) addBuildToolDefaults(config *ScanOptions, utils Utils) e
 			{Name: "go.dependencyManager", Value: "modules"},
 		},
 		"gradle": {
-			{Name: "ignoreSourceFiles", Value: true, Force: true},
-			{Name: "gradle.localRepositoryPath", Value: ".gradle", Force: false},
+			{Name: "gradle.resolveDependencies", Value: true, Force: true},
+			{Name: "gradle.ignoreSourceFiles", Value: true, Force: true},
+			{Name: "gradle.aggregateModules", Value: false, Force: true},
+			{Name: "gradle.runAssembleCommand", Value: true},
+			{Name: "gradle.runPreStep", Value: true},
+			{Name: "gradle.downloadMissingDependencies", Value: true},
+			{Name: "gradle.localRepositoryPath", Value: ".gradle"},
+			{Name: "excludes", Value: "**/*sources.jar **/*javadoc.jar"},
+			{Name: "ignoreSourceFiles", Value: false},
```

Comment:
```suggestion
			{Name: "ignoreSourceFiles", Value: true, Force: true},
			{Name: "includes", Value: "**/*.jar"},
```
The `ignoreSourceFiles` in general for all tools should be true and the `includes` should be similar to maven